src/interfaces.d.ts

```diff
@@ -85,7 +85,7 @@ export type SupportedClassName = string | null | undefined;
 
 export type DeferredVirtualProperties = (inserted: boolean) => VNodeProperties;
 
-export type FocusFunction = () => boolean;
+export type NodeOpFunction = () => boolean;
```

Comment:
Do you mean `NoOpFunction`?

Reply:
nah it's a `NodeOperationFunction`, perhaps I should expand that.

Reply:
how about including the word predicate? because it is one?